test/oli_web/live/remix_section_test.exs

```diff
@@ -1249,13 +1224,13 @@ defmodule OliWeb.RemixSectionLiveTest do
       assert view
              |> has_element?(
                ".remix_materials_table tbody tr:first-of-type td:nth-of-type(2)",
-               "Elixir Page"
+               "Another orph. Page"
              )
 
       assert view
              |> has_element?(
                ".remix_materials_table tr:last-of-type td:nth-of-type(2)",
-               "Another orph. Page"
+               "Elixir Page"
```

Comment:
With the suggested changes I think this should go back to how it was before
This should be the `first-of-type`.

Reply:
Updated accordingly